packages/libs/error-stack/src/report.rs

```diff
@@ -653,6 +632,11 @@ impl<Context> std::process::Termination for Report<Context> {
     }
 }
 
+// TODO:
+// we might want this to be implemented for `Report<Context>`
+// instead, in that case tho `Report<Context>` would be empty,
+// which would break an invariant, but using `Option<Report<Context>>` can be a bit unwieldy, maybe
+// we can have an alias?
```

Comment:
I think we simply don't want it yet. We could still add it at a later point if we find a good solution